testplan/testing/multitest/base.py

```diff
@@ -833,8 +850,25 @@ def _run_parallel_testcases(self, testsuite, execution_groups):
 
         # Add all non-empty parametrization reports into the list of returned
         # testcase reports, to be added to the suite report.
+        # Calculate runtime of the parametrized group as well
         for param_report in parametrization_reports.values():
             if param_report.entries:
+                start_times = []
```

Comment:
Here you repeat yourself, mostly, why are the checks more strict here than in the linear case?